src/jdk.sctp/unix/native/libsctp/SctpNet.c

```diff
@@ -144,10 +143,19 @@ sctpHandleSocketError(JNIEnv *env, jint errorValue)
             break;
     }
     errno = errorValue;
-    JNU_ThrowByNameWithLastError(env, xn, "NioSocketError");
+    if (message == NULL) {
+        JNU_ThrowByNameWithLastError(env, xn, "NioSocketError");
```

Comment:
"NioSocketError" is confusing as the failing message, maybe we should change that.